### PR TITLE
Add blackbox exporter

### LINKED ISSUE #1036: docs/components.md wrong? Blackbox exported not available
Hi,

I think `docs/components.md` maybe wrong. It states "Ingress and Service resources, which are probed using Prometheus Blackbox exporter", but there is no blackbox-exporter deployed. Instead there are statements like this in the manifests:
```
// TODO: add blackbox-exporter
```

floek

### PR DESCRIPTION
**Pull request by floek**
Hi,

I've added the blackbox exporter.
Fixes https://github.com/bitnami/kube-prod-runtime/issues/1036

Cheers,
floek

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi,

Thank you very much for the PR! I believe this should be part of a new BKPR release branch, as it is adding a new image. I will put the PR on hold until we have plans on working on this.

File: manifests/components/blackbox-exporter.jsonnet [hunk 1]
```diff
@@ -0,0 +1,107 @@
+// vim set syntax=jsonnet
+// Imports
+local kube = import "../vendor/github.com/bitnami-labs/kube-libsonnet/kube.libsonnet";
+local kubecfg = import "kubecfg.libsonnet";
+
+// Cluster-specific configuration
+local BLACKBOX_EXPORTER_IMAGE = "bitnami/blackbox-exporter:0.18.0-debian-10-r83";
```
Comment (javsalgar) [inline manifests/components/blackbox-exporter.jsonnet:7]: This should be in `manifests/components/images.json`, as that file is automatically updated by our CI/CD system.

File: manifests/components/blackbox-exporter.jsonnet [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+{
+  local this = self,
+
+  be_config:: {
```
Comment (javsalgar) [inline manifests/components/blackbox-exporter.jsonnet:12]: Could you explain this configuration?

**Review by javsalgar — CHANGES_REQUESTED** [review]

File: manifests/components/blackbox-exporter.jsonnet (hunk 1 above)
Comment (floek) [reply manifests/components/blackbox-exporter.jsonnet:7]: I see, I'll change this.

File: manifests/components/blackbox-exporter.jsonnet (hunk 2 above)
Comment (floek) [reply manifests/components/blackbox-exporter.jsonnet:12]: This is just the default config from https://github.com/prometheus/blackbox_exporter/blob/master/blackbox.yml.

File: manifests/components/blackbox-exporter.jsonnet (hunk 1 above)
Comment (floek) [reply manifests/components/blackbox-exporter.jsonnet]: done.

**Pull request closed without merging**